src/app/home/room-creation-modal/room-creation-modal.component.html

```diff
@@ -0,0 +1,33 @@
+<h1 mat-dialog-title>Create Room</h1>
+<div mat-dialog-content class="form-container">
+  <form [formGroup]="formGroup">
+    <mat-form-field appearance="legacy" >
+      <mat-label>Give your room a cool name</mat-label>
+      <input matInput formControlName="name">
+    </mat-form-field>
+
+    <mat-form-field appearance="legacy">
+      <mat-label>Pepole in conflicts</mat-label>
+      <input matInput required type="number" min="1" formControlName="numberOfPlayers">
+      <mat-hint>Don't count yourself</mat-hint>
+    </mat-form-field>
+
+    <mat-form-field *ngIf="formGroup.value.numberOfPlayers === 1"  appearance="legacy">
```

Comment:
`numberOfPlayers` shows unresolved in my IDE. Could you double check if this is correctly used?

Reply:
Yep it is correctly used, it shows that because formGroup.value is not the type of RoomCreationRequest until we cast it. I tried the (`fromGroup.value as RoomCreationRequest).value` in the HTML but it was complaining.
I can either create a method that returns the boolean value then I can do the cast in the TS file or you can just shut up you IDE by disabling that inspection like I did

Reply:
try [`$any()`](https://angular.io/guide/template-typecheck)

Reply:
here you go